### DISCUSSION TITLE
How do site settings work ?

### DISCUSSION THREAD

**Discussion by Norsagir** [mealie-recipes/mealie#331] [Q&A]
Hi !
First of all, thank you for this great tool I added to my NAS ! I didn't know that such a thing existed !

My question is : how do site settings work ? I mean, I'm not a dev so I can't look in the code, but what I understand is that settings are stored locally in the web browser ?
For example, the language is making me crazy ! I reset my Mealie data, reinstall the docker image from scratch, I configure French as the main language. When I'm consulting the website from another device (clean device with no cache), language is still English, so I must login and set French again, then the website will be in French for that browser.

Are settings such as language really stored in the web browser ?
I wanted to ask you this before I open a feature request asking for settings to be stored in the app so that everyone can see the site in French.

Thank you for your help :)

**Comment by sephrat** [comment]
Does that mean your issue was not fixed in v0.4.3? https://github.com/hay-kot/mealie/issues/315

**Reply by Norsagir** [reply]
This issue is fixed because now, when I click on French and save settings, site is in French.
The issue was about language setting that could not be changed at all.

Now, the site is in French but just for the web browser I used in order to apply settings.
For example, when I'm on the site with another PC and another browser (from Linux and Chrome to Windows and Firefox), the website is in English again.

I think this is a problem and not a normal behaviour since changes to default theme are applied to all web browser...
The issue may come from my web browsers, or the way I set the app, or maybe the code itself. I'm not sure about anything, that's why I wanted to ask before opening an issue.

**Reply by sephrat** [reply]
I've experienced this too on my own setup in the past but thought it was because of the linked issue. I still have this issue in 0.4.3 too, so you're not alone. :-) 
I'll take a look. Can you please create an issue so we track this?

**Reply by Norsagir** [reply]
Yes of course, I will open the issue as soon as I can :)
Thank you for your help !
